src/tileset/wangset/wang_tile.rs

```diff
@@ -0,0 +1,64 @@
+use std::str::FromStr;
+
+use xml::attribute::OwnedAttribute;
+
+use crate::{
+    error::Error,
+    util::{get_attrs, XmlEventResult},
+    Result, TileId,
+};
+
+/**
+The Wang ID, given by a comma-separated list of indexes (starting from 1, because 0 means _unset_) referring to the Wang colors in the Wang set in the following order: top, top right, right, bottom right, bottom, bottom left, left, top left (since Tiled 1.5). Before Tiled 1.5, the Wang ID was saved as a 32-bit unsigned integer stored in the format 0xCECECECE (where each C is a corner color and each E is an edge color, in reverse order).
+*/
+#[derive(Debug, PartialEq, Clone, Copy)]
+pub struct WangId(pub [u32; 8]);
+
+impl FromStr for WangId {
+    type Err = ();
```

Comment:
You probably saw this on some other code from the crate too, but it's not good practice to set the error to the unit type. I'd prefer if you made a unit struct `struct WangIdParseError;` which implemented `std::error::Error` instead

Reply:
Even better, should I use type Err = Error; from crate::error::Error? I could also add a new type of error on Error enum.

Reply:
If it is an error that can be thrown when loading maps, sure.